### PR TITLE
Release prep

### PR DESCRIPTION
**Pull request by mcsauder**
Hi Crayzee Wulf,

This PR adds a method to the SerialPort::Implementation class to compute the required time delta for an additional byte to arrive at the port, adds coverage for the ios good() method in Unit_Tests.cpp, and performs a great deal of formatting for uniformity throughout the library.

Unit tests pass (thanks to your work!), and things seem to be in great shape!

Please let me know if you have any questions on this PR.

-Mark

### CODE REVIEW

File: src/SerialStream.cpp [hunk 1]
```diff
@@ -38,40 +38,47 @@ namespace LibSerial
...
         }
     }
 
     void
     SerialStream::Open(const std::string& fileName,
                        const std::ios_base::openmode& openMode)
-    try {
-        // Create a new SerialStreamBuf if one does not exist. 
-        if (mIOBuffer == nullptr)
+    {
```
Comment (crayzeewulf) [inline src/SerialStream.cpp]: I see that you are not a fan of function-try-blocks. :) I have no issues putting the try block inside the function body. FYI, the reasons I tend to use function-try-blocks are (1) most importantly, to catch exceptions from constructor initializer lists, (2) they save one level of horizontal indentation and (3) they will not allow anyone to (accidentally) add any statements to the function after the catch(...) block. Exceptions thrown by such statements will not be caught within the function body.

Comment (mcsauder) [reply src/SerialStream.cpp]: Hi Crayzee Wulf, I can put that back the way you had it.  I was just trying to unify conventions.  I will update them throughout the library and push up the work shortly!

Comment (crayzeewulf) [reply src/SerialStream.cpp]: Mark: No need to undo them. They are perfectly fine as they are and are equivalent to function-try-blocks in this context.

Comment (mcsauder) [reply src/SerialStream.cpp]: Shoot!  Already done and pushed up! :)

File: test/UnitTests.cpp [hunk 2]
```diff
@@ -29,25 +29,6 @@
 using namespace LibSerial;
 
 LibSerialTest::LibSerialTest()
-    : numberOfTestIterations(10)
```
Comment (crayzeewulf) [inline test/UnitTests.cpp]: Glad you moved these to in-class-initialization! This is much cleaner now. 👍

File: src/SerialStream.cpp (hunk 1 above)
Comment (crayzeewulf) [reply src/SerialStream.cpp]: Even better. You are fast!

**Pull request merged**